service/src/main/java/org/apache/griffin/core/measure/entity/Rule.java

```diff
@@ -49,14 +45,18 @@ Licensed to the Apache Software Foundation (ASF) under one
     @NotNull
     private String dslType;
```

Comment:
I suggest set dslType as enum too, containing "spark-sql", "df-ops" and "griffin-dsl"